### ISSUE TITLE
Fix colour contrast on website

### ISSUE THREAD

**Issue by alexchen817** [alexchen817/alexchen817.github.io#24]
Currently, the colour contrast is too weird. Some of the text is not readable with my current colour scheme. I want to be able to make visitors enjoy their time on my website, and not have to squint to read or have a hard time reading information without getting distracted.

Criteria: 
choose a different colour scheme
pick a better font
reduce the speed of particles
make the particles less visible
